ajax/libs/color/0.11.3/index.js

```diff
@@ -0,0 +1,450 @@
+/* MIT license */
+var clone = require('clone');
+var convert = require('color-convert');
+var string = require('color-string');
```

Comment:
wait for reply https://github.com/cdnjs/cdnjs/issues/8531#issuecomment-244551903